src/app/screens/Calculator/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import {View, TextInput} from 'react-native';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+
+import styles from './styles';
+import Button from './components/Button';
+import { retrieveButtons } from './utils';
+
+const Calculator = () => {
+  //Variables y funcionesa
+  const [display, setDisplay] = useState('');
+
+  const leerPresionado = target => {
+    GET_BUTTONS.find(button => button.label === target).action();
+  };
+
+  let GET_BUTTONS = retrieveButtons(display, setDisplay);
+
+  //Fin de variables y funciones
+  return (
+    <KeyboardAwareScrollView style={styles.contain}>
+      <View style={styles.containerCalculadora}>
+        <View style={styles.boxPantalla}>
+          <View style={styles.pantalla}>
```

Comment:
Don't wrike spanglish. Our code base is all in english, so we should respect that.

Reply:
I will make the changes respecting that.